### PR TITLE
Favicon

### PR DESCRIPTION
**Pull request by sd0623**
- Added favicon (for every browser)
- Modified NavBar.tsx (clean code)
- Adjusted the 'Get Started' button when hovering
- Added .vscode to gitignore (to remove my env settings)

### CODE REVIEW

File: src/pages/NavBar.tsx [hunk 1]
```diff
@@ -252,6 +242,7 @@ function GetStarted() {
 }
 
 const before_pages = ['home', 'about', 'learn', 'solve']
+// TODO(SATH): After login navbar change
 // const after_pages = ['home', 'about', 'learn', 'solve', 'submit'];
```
Comment (cameroncuster) [inline src/pages/NavBar.tsx:246]: so what is this?

File: src/pages/NavBar.tsx [hunk 2]
```diff
@@ -116,7 +111,6 @@ interface pagesProps {
 
 function NavItems({ pages }: pagesProps) {
   const location = useLocation()
-  // const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null);
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (cameroncuster) [inline src/pages/NavBar.tsx:114]: why are these unused?

File: src/pages/NavBar.tsx [hunk 3]
```diff
@@ -116,7 +111,6 @@ interface pagesProps {
...
   const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null)
```
Comment (cameroncuster) [inline src/pages/NavBar.tsx:116]: do we need the `no-unused-vars` right below this line?

**Review by cameroncuster — APPROVED** [review]
LGTM

File: .gitignore [hunk 4]
```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.vscode
```
Comment (cameroncuster) [inline .gitignore:20]: I don't get why it committed your settings? like our setup must be different cause I don't think I have a file like this

File: src/pages/NavBar.tsx (hunk 1 above)
Comment (sd0623) [reply src/pages/NavBar.tsx:246]: We wanted different navbar options when user is logged in (submit for judge with user's own solution and I/O files)

File: .gitignore (hunk 4 above)
Comment (sd0623) [reply .gitignore:20]: when i added the formatOnSave for prettier vscode env settings created this, thats why i added the .vscode to gitignore

File: src/pages/NavBar.tsx (hunk 3 above)
Comment (sd0623) [reply src/pages/NavBar.tsx:116]: yeah i will look into this

File: src/pages/NavBar.tsx (hunk 2 above)
Comment (sd0623) [reply src/pages/NavBar.tsx:114]: will look into that

**Pull request merged**